tests/shouldwork/Cores/Xilinx/Floating/TH.hs

```diff
@@ -555,6 +555,42 @@ fromUBasicSamples = delayOutput0 (natToNum @F.FromU32DefDelay) $
     is = is0 ++ repeat (last is0)
     os = replicate d 0 ++ os0
 
+fromSBasicSamples :: [(Signed 32, Float)]
+fromSBasicSamples = delayOutput0 (natToNum @F.FromS32DefDelay) $
+  map (\x -> (x, fromIntegral x)) (specials ++ map (* (-1)) specials)
```

Comment:
Never mind! As you pointed out, `specials ++ map (* (-1)) specials` already covers it, and much better at that!